examples/gaussian_process/plot_gpr_prior_posterior.py

```diff
@@ -158,7 +158,7 @@ def plot_gpr_samples(gpr_model, n_samples, ax):
 )
 
 # %%
-# Periodic kernel
+# Exp-Sine-Squared kernel
```

Comment:
The ExpSineSquared kernel is a [periodic kernel](https://scikit-learn.org/stable/modules/gaussian_process.html#exp-sine-squared-kernel). Do you find it more helpful to say `Exp-Sine-Squared` here?

Reply:
Yes, I think it's better to be consistent. We could also add a sentence explicitly saying that these names are synonymous.